StereoKitC/xr_backends/extensions/ext_render_model.h

```diff
@@ -11,14 +11,14 @@
 namespace sk {
 
 typedef struct xr_render_model_t {
-	XrRenderModelIdEXT id;
-	XrRenderModelEXT   render_model;
-	model_t            model;
-	XrSpace            space;
-	model_node_id*     anim_nodes;
-	int32_t            anim_node_count;
-	XrRenderModelStateEXT state_query;
-};
+	XrRenderModelIdEXT    id;
+	XrRenderModelEXT      render_model;
+	model_t               model;
+	XrSpace               space;
+	model_node_id*        anim_nodes;
+	int32_t               anim_node_count;
+	XrRenderModelStateEXT state_query = { XR_TYPE_RENDER_MODEL_STATE_EXT };
```

Comment:
This violates some of the "Plain old Data" design I try to stick to in StereoKit, and I believe introduces an invisible constructor? I'd like to avoid this!

Reply:
Ahh okay! We'll want to change it here as well, which is where I was "inspired" 😄  

https://github.com/StereoKit/StereoKit/blob/b4a113fffd0693acb772b931add2f682c5419656/StereoKitC/xr_backends/extensions/hand_mesh.cpp#L27

Reply:
Whoops! Yeah, that one looks like it was probably an evolution from raw globals to a state struct! That shouldn't have survived 😅 Initialization like that can lead to issues when "restarting" StereoKit in the same process, since relying on global initialization means the variables don't always get reset properly!

Reply:
Updated in https://github.com/StereoKit/StereoKit/pull/1227